src/ops/info.rs

```diff
@@ -384,7 +383,7 @@ fn try_get_msrv_from_nearest_manifest_or_ws(
     ws: Option<&Workspace>,
 ) -> Option<semver::Version> {
```

Comment:
this should probably not be returning `semver::Version`.  At minimum, `PartialVersion`, maybe even `RustVersion`.  This also means that `config.load_global_rustc` should probably be loaded into a `PartialVersion`.  In fact, in the resolver, we do:

```rust
        let rust_version = if let Some(ver) = ws.rust_version() {
            ver.clone().into_partial()
        } else {
            let rustc = ws.gctx().load_global_rustc(Some(ws))?;
            let rustc_version = rustc.version.clone().into();
            rustc_version
        };
        version_prefs.max_rust_version(Some(rust_version));
```

Reply:
Make sense. I've updated it to use PartialVersion directly.